src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event extends Task {
```

Comment:
Maybe it would be a good idea to include class description? This way, fellow developers can catch a glimpse of the class purpose